modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -147,6 +148,7 @@ public void onWrite(Iterator<CommandClosure<WriteCommand>> iterator) {
      * Handler for the {@link UpdateCommand}.
      *
      * @param cmd Command.
+     * @param commandIndex Index of the RAFT command.
      */
     private void handleUpdateCommand(UpdateCommand cmd, long commandIndex) {
         storage.runConsistently(() -> {
```

Comment:
I see no place where you skip already applied commands

Reply:
Tried to fix it